pcbuilder_crawlers/spiders/afuture.py

```diff
@@ -59,5 +60,5 @@ def parse_item(self, response):
 
     def get_category(self, response):
         self.category = response.xpath("//*[@id='content']/h1/text()")\
-            .extract()
+            .extract()[0]
```

Comment:
Use join instead of first index. Using first index here can cause some bugs.
```
self.category = ''.join(response.xpath("//*[@id='content']/h1/text()").extract())
```